Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/brigadecore/brigade"
-  revision = "1eec8b07ca785154cf4df0b697378e205d7a4397"
+  revision = "509b6edd37222cade4b3970554088b2046b46a44"
```

Comment:
After we release Brigade 1.3, we should change this to point to that version.

Reply:
Agree, of course. I'll open an issue so we don't lose track of that.